src/pages/courses/courses.js

```diff
@@ -67,154 +74,939 @@ const useStyles = makeStyles(theme => ({
+              {`Are you sure you want to remove the course ${course.title}`}
+            </DialogTitle>
+
+            <DialogContent>
+              <DialogContentText id="alert-dialog-description">
+                When you remove a course you can't access the course anymore
+              </DialogContentText>
+            </DialogContent>
+
+            <DialogActions>
+              <Button
+                onClick={handleDialogClick}
+                className={classes.removeCourseDisagree}
+              >
+                Disagree
+              </Button>
+
+              <Button
+                onClick={() => {
+                  _removeCourse({
+                    client,
+                    query: removeCourseById,
+                    id: course.id,
+                    setCourseRemoved,
+                    setCourses,
+                    listAllCourses,
+                    enqueueSnackbar
+                  });
+                }}
+                className={classes.removeCourseAgree}
+                autoFocus
+              >
+                Agree
+              </Button>
+            </DialogActions>
+          </Dialog>
+
+          <IconButton
+            className={classes.updateIcon}
+            onClick={() =>
+              setUpdateCourseState({ state: !updateCourseState.state, course })
+            }
+          >
+            <UpdateIcon />
+          </IconButton>
+        </CardActions>
+
+        <Collapse in={expanded} timeout="auto" unmountOnExit>
+          <CardContent className={classes.cardContent}>
+            <Typography
+              className={`${classes.marginSvgIcon} ${classes.centerIconText}`}
+            >
+              <PermIdentityIcon />
+              {courseCreator.firstName
+                ? `${courseCreator.firstName} ${courseCreator.secondName}`
+                : "Loading"}
+            </Typography>
+
+            <Typography
+              className={`${classes.marginSvgIcon} ${classes.centerIconText}`}
+            >
+              <DescriptionIcon />
+              Complete description
+            </Typography>
+
+            {course.description
+              .split(/(\r\n|\n|\r)/gm)
+              .map((paragraph, index) => (
+                <Typography key={index} paragraph>
+                  {paragraph}
+                </Typography>
+              ))}
+          </CardContent>
+        </Collapse>
+      </Card>
+    </Grid>
+  );
+};
+
+const CreateCourse = ({
+  classes,
+  client,
+  setCreateCourseState,
+  setCourses,
+  createCourse,
+  createCourseState,
+  authentication
+}) => {
+  const { enqueueSnackbar } = useSnackbar();
+
+  const [privateCourse, setPrivateCourse] = useState(false);
+
+  const handlePrivateCourse = () => {
+    setPrivateCourse(!privateCourse);
+  };
+
+  return (
+    <Box
+      p={5}
+      bgcolor="white"
+      borderRadius="borderRadius"
+      className={classes.boxCreateCourse}
+    >
```

Comment:
Add a title inside the `Box` tag. Not that big :smile_cat:, this image is just an example.
Why not center the content in the `Box` component ? :thinking: 
![title](https://user-images.githubusercontent.com/12905770/74119690-4e136a80-4b9f-11ea-808e-b57326923ab7.jpg)

Reply:
Done, I have put the title (I have tried bold, but it doesn't integrate with the rest of the UI). I have also specified a max width and centered the items (full width).